flocker/node/agents/pd.py

```diff
@@ -0,0 +1,383 @@
+# -*- test-case-name: flocker.node.agents.functional.test_pd -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+A PD implementation of the ``IBlockDeviceAPI``.
```

Comment:
Possibly this module should be named "gce"? I know that's inconsistent but maybe it's the other modules that have bad names :grinning: